terraform/environments/mlra/ecs.tf

```diff
@@ -0,0 +1,75 @@
+#------------------------------------------------------------------------------
+# ECS
+#------------------------------------------------------------------------------
+
+module "windows-ecs" {
+
+  source = "github.com/ministryofjustice/modernisation-platform-terraform-ecs"
+
+  subnet_set_name         = local.subnet_set_name
+  vpc_all                 = local.vpc_all
+  app_name                = local.application_name
+  container_instance_type = local.application_data.accounts[local.environment].container_instance_type
+  environment             = local.environment
+  ami_image_id            = local.application_data.accounts[local.environment].ami_image_id
+  instance_type           = local.application_data.accounts[local.environment].instance_type
+  user_data = base64encode(templatefile("user_data.sh", {
```

Comment:
Perhaps putting the templatefile call in a local would tidy this bit up a bit

Reply:
user_data = base64encode(local.user_data)

locals {
	user_data = templatefile("", {
		the variables to render
	})
}